lib/workers/pr/index.ts

```diff
@@ -376,7 +381,7 @@ export async function ensurePr(
           prTitle,
           prBody,
           labels: config.labels,
-          platformOptions,
+          platformOptions: getPlafromPrOptions(config),
```

Comment:
Maybe just mutate this result here, e.g. overriding `gitlabAutomerge` to set it to false?